recipes/xgboost/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "xgboost" %}
+{% set version = "0.6a2" %}
+{% set sha256 = "ebc4e2bf8c8266212e342ff8ec4f6ae469e8c41a05d099b6778de8424ce32563" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - Makefile.patch  # [osx]
```

Comment:
Also would be good if could just send this patch upstream. Hard coding compiler versions like that is not great. Hopefully they will be receptive.